Eastern-Research-Group/mywaterway
# Feature/state national uses update

## Related Issues:
* https://app.breeze.pm/projects/100762/cards/3416231

## Main Changes:
* Updated the state national uses file to fix an issue with Arkansas data.
* Moved MP so the data is in alphabetical order by state abbreviation. 

## Steps To Test:
1. Navigate http://localhost:3000/state/AR/water-quality-overview
2. Make sure the "Fisheries" use shows up under the "Aquatic Life" tab and not the "Eating Fish" tab.
